app/controllers/concerns/authentication.rb

```diff
@@ -59,8 +57,7 @@ def rbac_allowed?
     return true if ActiveModel::Type::Boolean.new.cast(Settings.disable_rbac)
     return valid_cert_auth? if identity_header.cert_based?
 
-    @rbac_api ||= ::RbacApi.new(raw_identity_header)
-    @rbac_api.check_user
+    user.authorized_to?(Rbac::COMPLIANCE_FULL_ACCESS)
```

Comment:
@romanblanco Are we planning on using the `authorized_to?` method for other things in the future besides only checking for full Compliance access? Just curious! I see that there are a lot of tests covering this and they all look great

Reply:
@marleystipich2 Exactly. As the next step of enabling RBAC, we'll have defined resources and actions _(according to the document attached to the story)_ and test users permission using the `authorized_to?` method.

e.g.:
```
current_user.authorized_to?('compliance:custom_role:create')
```